### DISCUSSION TITLE
Best way to "ignore" observableRequiresReaction

### DISCUSSION THREAD

**Discussion by hitsthings** [mobxjs/mobx#3578] [Q&A] [answered]
Hi I'm new to mobx. I am trying to ensure strict use of it in my app so I've turned on all the linting and I'm going through each warning in the logs.

Some cases where I am accessing an observable are in static spots where it doesn't make sense to observe anything. For example, React Router has the concept of a "loader" which really just runs once per route change. In those cases, I want to explicitly suppress the console warning so I can move on to the next problem. Is there any way to do that?

I tried `untracked(() => ...)` but that doesn't do it. I guess I could make some kind of `withConfiguration({ observableRequiresReaction: false }, () => ....)` to override configuration for the life of that callback. ...is that the best way to go? Or is there something built-in for this?

**Comment by mweststrate** [comment, marked as answer]
Use action / runInAction

**Reply by hitsthings** [reply]
Thanks!
